terpnetwork/chain.json

```diff
@@ -103,18 +103,25 @@
           "type": "cometbft",
           "version": "0.37.2"
         }
+      },
+      {
+        "name": "v4.1.0",
```

Comment:
Can you add an entry for "next_version_name": to the previous version, indicating the next version as "v4.1.0"? And can you add two new entries to v4.1.0 for "proposal" and "height" to indicate the upgrade prop # and upgrade height? Also feel free to add a dummy blank "next_version_name" to v4.1.0. Refer to https://github.com/cosmos/chain-registry/blob/master/ununifi/chain.json as a good example.